ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/components/ReportJobs.tsx

```diff
@@ -1,7 +1,78 @@
 import React from 'react';
+import { Card, CardBody, Divider } from '@patternfly/react-core';
 
-function ReportJobs() {
-    return <div />;
+import {
+    ComplianceScanConfigurationStatus,
+    ComplianceScanSnapshot,
+} from 'services/ComplianceScanConfigurationService';
+import JobDetails from 'Containers/Vulnerabilities/VulnerablityReporting/ViewVulnReport/JobDetails';
+import ReportJobsTable from 'Components/ReportJobsTable';
+import ConfigDetails from './ConfigDetails';
+
+type ReportJobsProps = {
+    scanConfig: ComplianceScanConfigurationStatus | undefined;
+};
```

Comment:
Small readability question:

What do you think about `type ReportJobProps` to immediately precede `function ReportJobs`

Reply:
Yeah, that makes sense. We should keep the types close to the component for readability